src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -263,13 +257,14 @@ private void assertCommandFailure(String inputCommand, Class<? extends Throwable
     /**
      * A stub class to throw an {@code IOException} when the save method is called.
      */
-    private static class JsonItemListIoExceptionThrowingStub extends JsonItemListStorage<Person, JsonAdaptedPerson> {
```

Comment:
@ZoroarkDarkrai I am not sure if this class is being used? If i recall correctly it is only used in line 94, which is being removed

Reply:
Idk either? I was just removing person from AB3.

Reply:
better remove it? or would it be useful in the future?

Reply:
Maybe we leave it here first.